src/cmd-cloud-prune

```diff
@@ -324,7 +324,7 @@ def deregister_aws_amis(build, cloud_config, dry_run):
     for ami in amis:
         region_name = ami.get("name")
         ami_id = ami.get("hvm")
-        snapshot_id = ami.get("snapshot")
+        snapshot_id = ami.get("snapshot") or "detectFromAMI"
```

Comment:
just provide a default value to `.get()` and it will do what you want here.

```
>>> { 'name': 'abcdefg' }.get("name", 'default')
'abcdefg'
>>> { 'name': 'abcdefg' }.get("blank", 'default')
'default'
```

let's also add a comment here with a little more explanation on what we are doing.. i.e. `If we are dealing with an old manifest where the snapshot ID isn't stored then let's instruct ore to detect the snapshot ID from the AMI.`